playgrounds/angular/src/app/app.component.ts

```diff
@@ -2,8 +2,8 @@ import { Component } from '@angular/core'
 import { instantMeiliSearch } from '../../../../src'
 
 const searchClient = instantMeiliSearch(
-  'https://ms-9060336c1f95-106.saas.meili.dev',
-  '5d7e1929728417466fd5a82da5a28beb540d3e5bbaf4e01f742e1fb5fd72bb66'
+  'https://demo-steam.meilisearch.com/',
+  '90b03f9c47d0f321afae5ae4c4e4f184f53372a2953ab77bca679ff447ecc15c'
```

Comment:
Maybe we should think about hiding this key?

Reply:
It is the search key! We want to provide it to the users to have a direct functioning playground. I understand this might be a security issue but we prefer providing development comfort I think. @curquiza 

 @alallema  As this is not related to the PR, could you approve the pr if it is oke for you?